src/main/java/com/minecraftabnormals/abnormals_core/core/util/DataUtil.java

```diff
@@ -1,21 +1,23 @@
 package com.minecraftabnormals.abnormals_core.core.util;
 
-import net.minecraft.block.Block;
-import net.minecraft.block.Blocks;
-import net.minecraft.block.ComposterBlock;
-import net.minecraft.block.FireBlock;
+import net.minecraft.block.*;
```

Comment:
Don't import everything in `net.minecraft.block`

Reply:
Ugh, I really need to change IntelliJ settings.

Reply:
Fixed with latest commit.